sql_generators/active_users_aggregates_v3/templates/mobile_query.sql

```diff
@@ -125,43 +111,51 @@ baseline_with_searches AS (
       SAFE_CAST(NULLIF(SPLIT(baseline.normalized_os_version, ".")[SAFE_OFFSET(2)], "") AS INTEGER),
       0
     ) AS os_version_patch,
-    baseline.durations AS durations,
     baseline.submission_date,
-    baseline.uri_count,
-    baseline.is_default_browser,
+    metrics.uri_count,
+    metrics.is_default_browser,
     baseline.distribution_id,
     CAST(NULL AS string) AS attribution_content,
     CAST(NULL AS string) AS attribution_source,
     CAST(NULL AS string) AS attribution_medium,
     CAST(NULL AS string) AS attribution_campaign,
     CAST(NULL AS string) AS attribution_experiment,
     CAST(NULL AS string) AS attribution_variation,
-    search.ad_clicks,
-    search.organic_search_count,
-    search.search_count,
-    search.search_with_ads,
-    CAST(NULL AS FLOAT64) AS active_hours_sum
+    CAST(NULL AS FLOAT64) AS active_hours_sum,
+    is_daily_user,
+    is_weekly_user,
+    is_monthly_user,
+    is_dau,
+    is_wau,
+    is_mau
   FROM
     baseline
   LEFT JOIN
-    search_metrics search
-    ON search.client_id = baseline.client_id
-    AND search.submission_date = baseline.submission_date
+    metrics
+    ON baseline.client_id = metrics.client_id
+    AND baseline.normalized_channel IS NOT DISTINCT FROM metrics.normalized_channel
 ),
-baseline_with_searches_and_attribution AS (
+unioned_with_attribution AS (
   SELECT
-    baseline.*,
-    attribution_data.install_source,
-    attribution_data.adjust_network
+    unioned.*,
+    {% if app_name == "fenix" or  app_name == "firefox_ios" %}
```

Comment:
Will this account for, e.g. MozillaOnline? I guess we don't care about attribution I. That case?

Reply:
Offline, agreed  to not include it.